pydotorg/urls.py

```diff
@@ -1,71 +1,72 @@
-from django.conf.urls import include, url, handler404
```

Comment:
Nope... it's good as it stands. The problem was that the page not contain embedded social media. There's nothing to stop us linking to it. What we have is compliant with the BBC rules.